config/config.go

```diff
@@ -0,0 +1,54 @@
+package config
+
+import (
+	"fmt"
+)
+
+type Config struct {
+	BaseConfig `mapstructure:",squash"`
+
+	Panacea PanaceaConfig `mapstructure:"panacea"`
+	Enclave EnclaveConfig `mapstructure:"enclave"`
+}
+
+type BaseConfig struct {
+	LogLevel       string `mapstructure:"log-level"`
+	OracleMnemonic string `mapstructure:"doracle-mnemonic"`
+}
+
+type PanaceaConfig struct {
+	GRPCAddr string `mapstructure:"grpc-addr"`
+}
+
+type EnclaveConfig struct {
+	Enable                  bool   `mapstructure:"enable"`
+	AttestationProviderAddr string `mapstructure:"attestation-provider-addr"`
```

Comment:
I'm not sure if we need this setting.
Let's keep it for now and check later.

Reply:
Same to me.
I think the `Enable` is needed for running and testing the doracle in SGX-disabled environment.
And `AttestationProviderAddr` will be used for verification of remote report I think.
I'll make this clear when I add SGX related things